src/coreclr/vm/ilstubresolver.cpp

```diff
@@ -348,6 +348,43 @@ void ILStubResolver::SetLoaderHeap(PTR_LoaderHeap pLoaderHeap)
     m_loaderHeap = pLoaderHeap;
 }
 
+COR_ILMETHOD_DECODER* ILStubResolver::FinalizeILStub(ILStubLinker* sl)
+{
+    STANDARD_VM_CONTRACT;
+
+#ifndef DACCESS_COMPILE
+    _ASSERTE(!IsILGenerated());
+    _ASSERTE(sl != NULL);
+
+    UINT maxStack;
+    size_t cbCode = sl->Link(&maxStack);
+    DWORD cbSig = sl->GetLocalSigSize();
+
+    COR_ILMETHOD_DECODER* pILHeader = AllocGeneratedIL(cbCode, cbSig, maxStack);
+    BYTE* pbBuffer = (BYTE*)pILHeader->Code;
+    BYTE* pbLocalSig = (BYTE*)pILHeader->LocalVarSig;
+    _ASSERTE(cbSig == pILHeader->cbLocalVarSig);
+
+    size_t numEH = sl->GetNumEHClauses();
+    if (numEH > 0)
+    {
+        sl->WriteEHClauses(AllocEHSect(numEH));
+    }
+
+    sl->GenerateCode(pbBuffer, cbCode);
+    sl->GetLocalSig(pbLocalSig, cbSig);
+
+    // Store the token lookup map
+    SetTokenLookupMap(sl->GetTokenLookupMap());
+    SetJitFlags(CORJIT_FLAGS(CORJIT_FLAGS::CORJIT_FLAG_IL_STUB));
+
+    return pILHeader;
+#else
+    DacNotImpl();
+    return NULL;
+#endif // !DACCESS_COMPILE
```

Comment:
```suggestion
```
`#endif // !DACCESS_COMPILE` can be after `AllocGeneratedIL`. (The DACCESS_COMPILE ifdef can be deleted from `AllocGeneratedIL`.)